configuration/verify-source.yaml

```diff
@@ -0,0 +1,57 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Configuration for verify-source CodeBuild project
+
+Resources:
+  CodeBuildProject:
+    Type: AWS::CodeBuild::Project
+    Properties:
+      Name: verify-source
+      ServiceRole: !GetAtt CodeBuildRole.Arn
+      Environment:
+        Type: LINUX_CONTAINER
+        ComputeType: BUILD_GENERAL1_SMALL
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:2.0
+      Source:
+        Type: NO_SOURCE
+        BuildSpec:
+          !Sub
```

Comment:
Is this !Sub needed?  What's being substituted exactly?  It doesn't seem like it could be the env vars.  e.g. $COMMIT_ID could not even possibly be known at the point the !Sub is evaluated.

If you remove it, you ought to be able to use a nicer form of YAML multi-line string.  Does that change the situation with regards to embedding the GPG keys?

Reply:
Nope, that sub function was left over from experimentation. Thanks.

The YAML string looks better now but I don't think the public key issue is helped at all. The whitelist string would still need to be formatted very specifically in order for gpg to recognize it.